<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CRulesView</class>
 <widget class="QWidget" name="CRulesView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Rule &amp;Name:</string>
     </property>
     <property name="buddy">
      <cstring>filter</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="3">
    <widget class="QToolButton" name="deleteRule">
     <property name="text">
      <string>Delete Rule</string>
     </property>
     <property name="icon">
      <iconset resource="MainWindow.qrc">
       <normaloff>:/resources/trash.png</normaloff>:/resources/trash.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QToolButton" name="enableRule">
     <property name="text">
      <string>Enable Rule</string>
     </property>
     <property name="icon">
      <iconset resource="MainWindow.qrc">
       <normaloff>:/resources/enable.png</normaloff>:/resources/enable.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="filter">
     <property name="placeholderText">
      <string>&lt;filter&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" rowspan="6" colspan="3">
    <widget class="QTreeView" name="rules">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QToolButton" name="disableRule">
     <property name="text">
      <string>Disable Rule</string>
     </property>
     <property name="icon">
      <iconset resource="MainWindow.qrc">
       <normaloff>:/resources/disable.png</normaloff>:/resources/disable.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>rules</tabstop>
 </tabstops>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
